src/stryker4s/index.pug

```diff
@@ -9,7 +9,7 @@ block content
 
         Currently Stryker4s is still in early development, which means it's propably not ready for large projects just yet.
 
-        But, this shouldn't hold you back to give Stryker4s a try. 
+        But this shouldn't hold you back to give Stryker4s a try. 
```

Comment:
I don't think the comma is needed here. Maybe we should change it to "However, this shouldn't hold you back to give Stryker4s a try."

Reply:
Sounds great